x/profiles/keeper/alias_functions.go

```diff
@@ -101,6 +122,26 @@ func (k Keeper) IterateUserApplicationLinks(ctx sdk.Context, user string, fn fun
 	}
 }
 
+// IterateUserRelationships iterates through the relationships with the given user address and performs the provided function
+func (k Keeper) IterateBlockedUsersByUserSubSpace(ctx sdk.Context, user string, subspace string, fn func(index int64, blocks types.UserBlock) (stop bool)) {
```

Comment:
This function is used only inside `GetUserBlockByBlockerSubspace` which should be removed since it's unused, so we can remove this as well I think